tests/integration/targets/eos_eapi/tests/cli/configure.yaml

```diff
@@ -19,10 +19,14 @@
       - show management api http-commands | json
     provider: '{{ cli }}'
 
-- name: Expect endpoint ports to be set
+- name: Expect http port to remain as 80 and not change to 81 ( with http disabled )
   assert:
     that:
       - http_config.stdout[0].httpServer.port == 80
+
+- name: Expect endpoint other ports to be set as configured
+  assert:
```

Comment:
I don't know that there is much value in breaking this down into two separate assert statements. A longer name or even just a comment before the assert task would be fine for me